setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = ['Click>=6.0', 'flake8-pytest-mark', 'flake8', 'flake8-json', 'six']
```

Comment:
You might want to pin a little more aggressively on `flake8-pytest-mark` considering how dependent you are on the interface.